### PR TITLE
Added documentation for Sector Alarm component

### PR DESCRIPTION
**Pull request by peroyvind**
**Description:**
Documentation for the new Sector Alarm component

## Checklist:

- [X] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [X] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/sectoralarm.markdown [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category:
+  - Alarm
+  - Sensor
+ha_release: 0.91
+ha_iot_class: Cloud Polling
+redirect_from:
```
Comment (klaasnicolaas) [inline source/_components/sectoralarm.markdown]: You can remove line 15 - 17

File: source/_components/sectoralarm.markdown [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+```yaml
+sector_alarm:
+  email: your@email.com
+  password: Password
+  alarm_id: 012345678
+  code: 1234
+```
+
+{% configuration %}
```
Comment (klaasnicolaas) [inline source/_components/sectoralarm.markdown:37]: Please remove the whitespaces between the variables.

File: source/_components/sectoralarm.markdown [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+Go to [minside.sectoralarm.no/](https://minside.sectoralarm.no/) and login to find your panel id. After you login it should be a part of the URL. Example: https://minside.sectoralarm.no/#!/systems/**012345678**
+
+## {% linkable_title Configuration %}
+
+To add Sector Alarm sensors and alarm to your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+sector_alarm:
+  email: your@email.com
```
Comment (klaasnicolaas) [inline source/_components/sectoralarm.markdown]:
```suggestion
  email: EMAIL_ADDRESS
```

File: source/_components/sectoralarm.markdown [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+  password: Password
```
Comment (klaasnicolaas) [inline source/_components/sectoralarm.markdown]:
```suggestion
  password: PASSWORD
```

File: source/_components/sectoralarm.markdown [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+  alarm_id: 012345678
+  code: 1234
```
Comment (klaasnicolaas) [inline source/_components/sectoralarm.markdown]:
```suggestion
  code: ALARM_CODE
```

File: source/_components/sectoralarm.markdown [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+
+## {% linkable_title Configuration %}
+
+To add Sector Alarm sensors and alarm to your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+sector_alarm:
+  email: your@email.com
+  password: Password
+  alarm_id: 012345678
```
Comment (klaasnicolaas) [inline source/_components/sectoralarm.markdown]:
```suggestion
  alarm_id: ALARM_ID
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

**Review by klaasnicolaas — APPROVED** [review]
✅ Can be merged after merging the parent PR 👍

**Comment by klaasnicolaas** [comment]
Parent PR still open

**Comment by frenck** [comment]
Closing PR, since the parent PR has been closed.

**Pull request closed without merging**